src/Event/DynamoDb/DynamoDbRecord.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+namespace Bref\Event\DynamoDb;
+
+final class DynamoDbRecord
+{
+    /** @var array */
+    private $record;
+
+    /**
+     * @param mixed $record
+     */
+    public function __construct($record)
+    {
+        if (! is_array($record) || ! isset($record['eventSource']) || $record['eventSource'] !== 'aws:dynamodb') {
+            throw new \InvalidArgumentException('Event source must come from DynamoDB');
+        }
+
+        $this->record = $record;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getKeys()
+    {
+        return $this->record['dynamodb']['Keys'];
+    }
+
+    /**
+     * @return mixed
```

Comment:
Same here, are the "images" always arrays?

Reply:
It will not always be an array depending on if the record is an update or delete.